project/src/js/ui/tournament/list/tournamentListView.js

```diff
@@ -0,0 +1,76 @@
+import h from '../../helper';
+import {header, empty, pad} from '../../shared/common';
+import layout from '../../layout';
+import i18n from '../../../i18n';
+import m from 'mithril';
+import tabs from 'polythene/tabs/tabs';
+
+export default function view(ctrl) {
+  const bodyCtrl = tournamentListBody.bind(undefined, ctrl);
+
+  return layout.free(header.bind(undefined, i18n('tournaments')), bodyCtrl, empty, empty);
+}
+
+const TABS = [{
+    id: 'started',
+    label: 'In Progress'
+}, {
+    id: 'created',
+    label: 'Upcoming'
+}, {
+    id: 'finished',
+    label: 'Completed'
+}];
+
```

Comment:
Here prefer this syntax:

```
function tabNavigation(currentTabFn) {
   return ...
}
```

Using arrow function is good for anonymous functions and expressions but not to declare a function.